auth_matrix/admin_urls.py

```diff
@@ -1,10 +1,14 @@
+from django.contrib import admin
 from django.urls import path
+
 from .admin_views import AuthorizationMatrixView
 
+app_name = "auth_matrix"
+
 urlpatterns = [
     path(
         "matrix",
-        AuthorizationMatrixView.as_view(),
+        admin.site.admin_view(AuthorizationMatrixView.as_view()),
         name="authorization_matrix",
     ),
 ]
```

Comment:
IMO this should be part of the user admin mixin and override the `AdminSite.get_urls` method to add additional URLs, instead of having to hook this URL up manually in your urlconf.